### PR TITLE
Added tomcat config for internalProxies

### LINKED ISSUE #1360: New UI file upload fails due to internal proxies
**Describe the bug**
When running the new UI in certain environments, the file upload feature fails due to the default tomcat internal proxies.

**To Reproduce**

1. Install 2019.1 in AWS
2. Enable the new UI
3. Try to upload a file in a contribution
4. Error occurs

This is fixed by configuring tomcat's internal proxies.

### PR DESCRIPTION
**Pull request by cbeach47**
##### Checklist
- [x] the [contributor license agreement][] is signed
- [x] commit message follows [commit guidelines][]
- [ ] documentation is changed or added - I will update the docs repo with the config once this is reviewed/merged.

##### Description of change
Enables the config `tomcat.internalProxies` (  #1360 )

[contributor license agreement]: https://www.apereo.org/node/676
[commit guidelines]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

**Comment by cbeach47** [comment]
@edalex-ian  - I don't have visibility into the `edalexEquella` build.  Can you take a quick look and see what might be up?

**Comment by PenghaiZhang** [comment]
> @edalex-ian - I don't have visibility into the `edalexEquella` build. Can you take a quick look and see what might be up?

Hi @cbeach47 , AWS failed due to a flaky test. I restarted it and let's see the result.

File: Source/Plugins/Core/com.equella.core/src/com/tle/tomcat/guice/TomcatModule.java [hunk 1]
```diff
@@ -39,6 +39,7 @@ protected void configure() {
       bindBoolean("tomcat.useBio");
       bindBoolean("sessions.neverPersist");
       bindBoolean("userService.useXForwardedFor");
+      bindProp("tomcat.internalProxies");
```
Comment (PenghaiZhang) [inline Source/Plugins/Core/com.equella.core/src/com/tle/tomcat/guice/TomcatModule.java]: Do we need to add `tomcat.internalProxies` with a default value to `optional-config.properties` ? 
I found "jvmroute.id" and "userService.useXForwardedFor"  already exist in the config file but "tomcat.useBio" and "sessions.neverPersist" do not.

Comment (cbeach47) [reply Source/Plugins/Core/com.equella.core/src/com/tle/tomcat/guice/TomcatModule.java]: Fair point - seems to be inconsistent in how the configs are documented.  I added this to the next CDM agenda.  For now, I'm leaning towards just merging as-is.

**Comment by cbeach47** [comment]
Thanks for the build restart @PenghaiZhang

**Comment by cbeach47** [comment]
Thanks for the tip on the auto-closing keyword.

**Pull request merged**